samples/TOML/filenames/Cargo.lock

```diff
@@ -0,0 +1,4 @@
+[[package]]
```

Comment:
Could you replace this sample with one of the "real" samples you linked to in your post?

Reply:
Sure. That one was generated from the [`hello_cargo`](https://doc.rust-lang.org/book/second-edition/ch01-02-hello-world.html) example. Sorry, I misunderstood what the PR template meant with the nesting of the sample item. I thought it that you had to have a sample, and the sources were more in line with the search verification.